### ISSUE TITLE
new PCA show method unwieldy when more than 10 PC

### ISSUE THREAD

**Issue by pdimens** [JuliaStats/MultivariateStats.jl#186]
Thank you for your hard work, I really like the new PCA show method. However, if there are more than 10 PC's, there is wild overflow in the REPL. Is it possible to elide the resulting tables (like how PrettyTables does it) to fit the REPL window?

![image](https://user-images.githubusercontent.com/19176506/156823598-8c2f7978-5289-4076-976d-7fc98806c7b1.png)

**Comment by wildart** [comment]
Yep, the compact output would be a nice thing. Meanwhile, try to use `;` at the end of  the expression to suppress output.
